common/templates/general.go

```diff
@@ -788,6 +788,13 @@ func in(l interface{}, v interface{}) bool {
 							return true
 						}
 					}
+				case reflect.Uint, reflect.Uint8, reflect.Uint16, reflect.Uint32, reflect.Uint64:
```

Comment:
It might be easier to use `lvv.CanUint()` (resp. `lvv.CanFloat()`, `lvv.CanInt()` here instead. (Note that this will require rewriting the switch as `switch { /* ... */ }` instead of a switch on `lvv.Kind()`.)

Reply:
Pushed the requested changes; the switch statement now uses the respective type's `Can*`.

I also went ahead and changed some prior checks around, decreasing indentation. While I was at it I satisfied `gopls/modernize` by modernising the for loop. Seems to still work as intended.